pkg/controller/secretsyncer/secretsyncer.go

```diff
@@ -188,6 +188,19 @@ func (r *reconciler) mirrorSecret(ctx context.Context, log *logrus.Entry, cluste
 		}
 		data[k] = source.Data[k]
 	}
+
+	if err := c.Get(ctx, types.NamespacedName{Name: to.Namespace}, &corev1.Namespace{}); err != nil {
```

Comment:
Can we instead add a config option to limit the target clusters? Users are not supposed to use this to create namesapces. The secret in question is only needed for app.ci

Reply:
Ack. Will do something like cluster group.

Reply:
Hu.
It has been implemented.
https://github.com/openshift/ci-tools/blob/b0351c25136f3e2695d5f2364dada541010230fe/pkg/controller/secretsyncer/config/config.go#L42

Reply:
/close

https://github.com/openshift/release/pull/16036